models/sale_artwork.coffee

```diff
@@ -58,6 +58,9 @@ module.exports = class SaleArtwork extends Backbone.Model
     count += if n is 1 then '' else 's'
     count
 
+  formatBidCount: ->
+    if @bidCount() is '0 bids' then '' else "(#{@bidCount()})"
```

Comment:
Ah! Yeah, `bidCount` is used below in `formatBidsAndReserve`, a method that does further string manipulation based on the bid count in a similar fashion https://github.com/artsy/microgravity/blob/master/models/sale_artwork.coffee#L47-L48

But since `bidCount` isn't displayed anywhere anymore, I did a little refactoring so we can just check the integer value in these little display logic methods. :)